src/metrics.rs

```diff
@@ -2604,61 +2599,73 @@ impl GooseAttack {
         };
     }
 
-    /* @FIXME:
     // Write an HTML-formatted report, if enabled.
     pub(crate) async fn write_html_report(
         &mut self,
         goose_attack_run_state: &mut GooseAttackRunState,
     ) -> Result<(), GooseError> {
         // Only write the report if enabled.
         if let Some(report_file) = goose_attack_run_state.report_file.as_mut() {
+            let test_start_time = self.metrics.history.first().unwrap().timestamp;
```

Comment:
I try to use `.expect()` instead of `.unwrap()` to more explicitly document our assumptions -- but not a blocker as it will panic either way.